middleware/authMiddleware.go

```diff
@@ -2,31 +2,49 @@ package middleware
 
 import (
 	"fmt"
+	"log"
+
+	"github.com/bokoness/lashon/models"
 	"github.com/bokoness/lashon/services"
 	"github.com/gofiber/fiber/v2"
 )
 
-func AuthMiddleware(c *fiber.Ctx) error {
+func InsertUserToSession(c *fiber.Ctx, user *models.User) int {
+	sess, err := services.GetStore(c)
+	statusCodeError := fiber.StatusInternalServerError
+	if err != nil {
+		return statusCodeError
+	}
+
+	sess.Set("user", user)
+
+	if err = sess.Save(); err != nil {
+		log.Fatal(fmt.Printf("this is the error %s", err))
+		return statusCodeError
+	}
+
+	return 0
```

Comment:
Why do you return 0? 
if you want to make sure that there is no error, you should return an error instead, and `nil` if there are no errors.
Then in when you call this method you do something like this:
```go
err := InsertUserToSesstion(c, user) 
if err != nil {
  return fiber.NewError(statusCodeError)
 }
 ```